tests/acceptance/helpers/sharingHelper.js

```diff
@@ -150,8 +156,14 @@ module.exports = {
         return res.ocs.data
       })
   },
+  getAllSharesSharedWithUser: function (user) {
+    return this.getAllShares(user, true)
+  },
+  getAllSharesSharedByUser: function (user) {
+    return this.getAllShares(user, false)
```

Comment:
Default is false, so saving a few characters.
```suggestion
    return this.getAllShares(user)
```